WalletWasabi.Fluent/Helpers/NotificationHelpers.cs

```diff
@@ -37,11 +37,12 @@ public static void Show(string title, string message, Action? onClick = null)
 			}
 		}
 
-		public static void Show(ProcessedResult result, Action onClick)
+		public static void Show(string walletName, ProcessedResult result, Action onClick)
 		{
 			if (TryGetNotificationInputs(result, out var title, out var message))
 			{
-				Show(title, message, onClick);
+				var notificationTitle = $"{walletName} - {title}";
```

Comment:
![aaaaaa](https://user-images.githubusercontent.com/52379387/142837437-36c83c2e-cffd-478d-9c26-c2037482fe8b.PNG)

Maybe we should put the title first?

Reply:
yeah, it seems weird indeed. I guess we could say only the wallet name in the title and extend the message to be more user friendly.

```
Testnet Wallet
0.0010 1565 BTC has been spent.
```
```
Daemon-nopass
0.0010 1387 BTC has been received.
```

What do you think?

Reply:
Yes it seems better.

But notice that the amount spent includes the fees. For me it would better if we just say the amount sent to the recipient, I guess that is a better UX.

```
Testnet Wallet
0.0010 0000 BTC has been "sent".
```

```
Daemon-nopass
0.0010 1387 BTC has been received.
```

If that is complicated then let's go with your suggestion.

Reply:
> But notice that the amount spent includes the fees. For me it would better if we just say the amount sent to the recipient, I guess that is a better UX.

I am afraid the users might be confused because they will see a different amount in the history if we would subtract the fee.

Reply:
Yes I get that, but still think it is better to show the amount the user intended to send, maybe this is just my preference.

Reply:
You convinced me. It will only show the sent amount.